app/src/main/java/fr/free/nrw/commons/contributions/UploadCountClient.java

```diff
@@ -14,10 +14,10 @@
 import fr.free.nrw.commons.concurrency.ThreadPoolExecutorService;
 import timber.log.Timber;
 
-public class UploadCountClient {
+class UploadCountClient {
     private ThreadPoolExecutorService threadPoolExecutor;
 
-    public UploadCountClient() {
+    UploadCountClient() {
```

Comment:
There's only a single usage of `UploadCountClient` - line 271 of `ContributionsActivity` - and that is in the same package so I tightened up the visibility of the class + constructor.  Happy to reverse the change if you prefer.